war/src/main/less/abstracts/theme.less

```
@@ -326,8 +327,8 @@
 
   each(@semantics, {
     --@{key}-color: @value;
-  })
-}
+  });
+ }
```

```suggestion
}
```

(to match the previous suggestion)